src/dataset/domain/models/Dataset.ts

```diff
@@ -1,25 +1,68 @@
-import { LabelSemanticMeaning } from './LabelSemanticMeaning.enum'
+export enum LabelSemanticMeaning {
+  DATASET = 'dataset',
+  FILE = 'file',
+  SUCCESS = 'success',
+  INFO = 'info',
+  WARNING = 'warning',
+  DANGER = 'danger'
+}
 
 export interface DatasetLabel {
   semanticMeaning: LabelSemanticMeaning
   value: string
 }
 
+export type DatasetMetadataSubField = Record<string, string>
+
+export const ANONYMIZED_FIELD_VALUE = 'withheld'
+type AnonymizedField = typeof ANONYMIZED_FIELD_VALUE
+
+export type DatasetMetadataField = string | DatasetMetadataSubField[] | AnonymizedField
+
+export type DatasetMetadataFields = Record<string, DatasetMetadataField>
+
+export enum MetadataBlockName {
+  CITATION = 'citation',
+  GEOSPATIAL = 'geospatial',
+  ASTROPHYSICS = 'astrophysics',
+  BIOMEDICAL = 'biomedical',
+  CODE_META = 'codeMeta20',
+  COMPUTATIONAL_WORKFLOW = 'computationalworkflow',
+  CUSTOM_ARCS = 'customARCS',
+  CUSTOM_CHIA = 'customCHIA',
+  CUSTOM_DIGAAI = 'customDigaai',
+  CUSTOM_GSD = 'customGSD',
+  CUSTOM_MRA = 'customMRA',
+  CUSTOM_PSI = 'customPSI',
+  CUSTOM_PSRI = 'customPSRI',
+  CUSTOM_HBGDKI = 'custom_hbgdki',
+  JOURNAL = 'journal',
+  SOCIAL_SCIENCE = 'socialscience'
+}
+
+export interface DatasetMetadataBlock {
+  name: MetadataBlockName
+  fields: DatasetMetadataFields
+}
+
 export interface DatasetField {
```

Comment:
Can we replace the DatasetField interface used in DataSummary with DatasetMetadataField?  I think that could work since the fields in the summary come from the list of metadata fields, and it would simplify the domain model

Reply:
Sounds good! I made the changes

Reply:
The changes look good, thanks!  

In retesting, I noticed a spacing issue with anonymized fields - fields that are anonymized are spaced closer together. Since this is a small issue that only happens with anonymized fields, should we create a separate issue for it, in order to merge this PR?
<img width="1167" alt="Screenshot 2023-05-24 at 11 37 29 AM" src="https://github.com/IQSS/dataverse-frontend/assets/675224/9f131a4e-b1bf-4a45-a420-b736c857dc20">

<img width="1183" alt="Screenshot 2023-05-24 at 11 37 18 AM" src="https://github.com/IQSS/dataverse-frontend/assets/675224/9597bf4d-acf0-4654-a33a-d0e2e73c9bbc">

Reply:
wow, that one was hard to notice, I think that's because the anonymized fields don't use the markdown, so they need a <p> tag to simulate the markdown. I made the change and I think now there is the same spacing

Reply:
looks good!